### PR TITLE
DDCNL-2688 | Remove use of deprecated SessionKeys attributes

### PR DESCRIPTION
**Pull request by jpatton638**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed

### CODE REVIEW

**Review by oc-ste — APPROVED** [review]

File: app/views/main.scala.html [hunk 1]
```diff
@@ -55,11 +55,11 @@
 @isUserResearchBannerHidden = @{request.cookies.exists((x: Cookie) => x.name == "mdtpurr")}
 
 @authProvider = @{
-    request.session.get(SessionKeys.authProvider)
+    request.session.get(TaiConstants.AuthProvider)
```
Comment (oc-ste) [inline app/views/main.scala.html:58]: @vimalsuresh need to check we can log in via verify and gg fine and that the sign out redirects you to ida/signout and gg/signout

File: app/views/main.scala.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
 }
 
 @isVerify = @{
-    authProvider.contains("IDA")
+    authProvider.contains(TaiConstants.AuthProviderVerify)
```
Comment (oc-ste) [inline app/views/main.scala.html:72]: @vimalsuresh need to check the correct menu bar is shown but I believe this is under test via ATs

**Review by oc-ste — APPROVED** [review]

**Review by oc-ste — APPROVED** [review]

File: app/views/main.scala.html (hunk 1 above)
Comment (vimalsuresh) [reply app/views/main.scala.html]: Tested in local , looks good

File: app/views/main.scala.html (hunk 2 above)
Comment (vimalsuresh) [reply app/views/main.scala.html]: Have AT's to test menu links, looks good

**Pull request merged**